content/reference-rippled.md

```diff
@@ -2698,7 +2695,7 @@ The response follows the [standard format](#response-formatting), with a success
 | `ledger_index_max` | Integer                                    | The sequence number of the most recent ledger actually searched for transactions. |
 | `limit`            | Integer                                    | The `limit` value used in the request. (This may differ from the actual limit value enforced by the server.) |
 | `marker`           | [(Not Specified)](#markers-and-pagination) | Server-defined value indicating the response is paginated. Pass this to the next call to resume where this call left off. |
-| `offset`           | Integer                                    | The `offset` value used in the request. |
+| `offset`           | Integer                                    | Included only in the JSON-RPC response. `offset` is a field from a deprecated legacy variation of the `account_tx` method. We recommend that you do not include the deprecated `offset` field in your request. When you don't include `offset` in your request, the `offset` value is returned in the response, but is always set to `0`. If you include an `offset` value in your request, it is returned as the `offset` value in the response. |
```

Comment:
@mDuo13 - This is absolutely awkward, but it is because I'm explaining some awkward functionality. Sure, blame the functionality... If `offset` is not sent in the request, it should not be returned in the response, but it is. 

How about this less verbose, but slightly more mysterious version?

> This legacy field is included in the JSON-RPC response only. It is always set to `0`.